### DISCUSSION TITLE
Time::Piece module requirement

### DISCUSSION THREAD

**Discussion by beldmit** [openssl/openssl#24574] [Q&A] [answered]
Is it OK that we require Time::Piece Perl module for tests?
I came across it in #24573

**Comment by nhorman** [comment]
I would think it would be fine, yes, though I'm trying to understand how you didn't hit this previously.  Time::Piece was added to tconversion.pl in august of 2023, so its been there for some time.  Is your systems installation a new install?

**Reply by beldmit** [reply]
No, I just didn't run upstream tests for a while. If it's a Perl core module then it's completely OK to me

**Reply by nhorman** [reply, marked as answer]
Ok, then given that its a core module, it seems like its fine to include
